app/models/document.rb

```diff
@@ -47,9 +47,14 @@ def promoted
     finder.links["ordered_related_items"].any? { |item| item["base_path"] == path }
   end
 
+  def truncated_description
+    # This truncates the description at the end of the first sentence
+    description.gsub(/\.\s[A-Z].*/, '.') if description.present?
+  end
+
 private
 
-  attr_reader :link, :rummager_document, :finder, :description, :facet_content_ids
+  attr_reader :link, :rummager_document, :finder, :facet_content_ids, :description
```

Comment:
If you use a tool like [Fork](https://git-fork.com/) it's possible to discard changes, rather than trying to revert them manually.